test/UserTest.php

```diff
@@ -1,6 +1,7 @@
 <?php
-namespace Icepace\Test;
-require_once '../vendor/autoload.php';
+
+require_once 'vendor/autoload.php';
```

Comment:
why did this change from '../vendor/autoload.php'?

Reply:
Corrected to ../vendor/autoload.php